internal/jj/commands.go

```diff
@@ -16,11 +16,11 @@ const (
 type CommandArgs []string
 
 func ConfigListAll() CommandArgs {
-	return []string{"config", "list", "--color", "never", "--include-defaults"}
+	return []string{"config", "list", "--color", "never", "--include-defaults", "--ignore-working-copy"}
 }
 
 func Log(revset string, limit int) CommandArgs {
-	args := []string{"log", "--color", "always", "--quiet"}
+	args := []string{"log", "--color", "always", "--quiet", "--ignore-working-copy"}
```

Comment:
@vic I had to revert this part because we want to snapshot when `log` is called. Otherwise, the revisions that have changes in it show as empty on start up, or when revset is changed.